lib/libzpool/zrepl_mgmt.c

```diff
@@ -457,6 +459,24 @@ uzfs_zinfo_store_last_committed_degraded_io_no(zvol_info_t *zinfo,
 	    DEGRADED_IO_SEQNUM, io_seq);
 }
 
+uint8_t
+uzfs_get_quorum(zvol_state_t *zv)
```

Comment:
lets make this as uzfs_zinfo_get_quorum and uzfs_zinfo_set_quorum and take 'zinfo' as argument rather than zvol_state_t, so that, in these APIs, we can take zinfo->main_zv and return the quorum value of main_zv.